README.md

```diff
@@ -1 +1,67 @@
-# cockroachdb-manifests
\ No newline at end of file
+# cockroachdb-manifests
+
+This is a Kustomization base for deploying CockroachDB to a Kubernetes cluster. The base depends on Cloudflare's
+[cfssl](https://github.com/cloudflare/cfssl) as a Certificate Authority for signing certificates for
+securing communication between nodes and clients.
+
+#### CFSSL
+
+"CFSSL is CloudFlare's PKI/TLS swiss army knife". This base requires cfssl and depends on the API server
+to sign certificates and retrieve the Certificate Authority it will trust. CFSSL provide a
+[docker container](https://hub.docker.com/r/cfssl/cfssl/) which can be deployed in Kubernetes.
+
+#### Certificates
+
+- The base relies on this [container](https://github.com/utilitywarehouse/docker-cockroach-cfssl-certs) it is used
+  as an init container to sign certificates on startup.
```

Comment:
Minor point, but the wording seems slighty off here. Maybe something like this instead?

```
- The base relies on [docker-cockroach-cfssl-certs](https://github.com/utilitywarehouse/docker-cockroach-cfssl-certs). It is executed via an init container, acquiring certificates on pod start.
```